charts/pega/README.md

```diff
@@ -208,7 +208,18 @@ Example:
 requestor:
   passivationTimeSec: 900
 ```
+### Security context
 
+You may configure security context for Pega pod deployments. By default ```pegasystems/pega``` image uses ```pegauser(9001)``` as the user.
+
+Example:
+
+```yaml
+tier:
+  - name: my-tier
+    securityContext:
+      runAsUser: RUN_AS_USER
```

Comment:
do you want to add a comment about that user this is? what I don't see is how security is related to the RUN_AS_USER parameter. ;-/

Reply:
you can address these documentation changes later using BUG-622868. All set!